### ISSUE TITLE
Every frame is a "key frame" after using detector for pre-annotation

### ISSUE THREAD

**Issue by aschernov** [opencv/cvat#349]
Hi!After using detector for pre-annotation every frame became the "key frame" Thus, it's very hard to  fix any attributes or change some object's properties because we need to do it on each frame manually. It will be great If we'll have some quick and simple way how to fix it or maybe opportunity to delete automatically key frames and set correct new ones.

**Comment by nmanovic** [comment]
In general it is possible to use re-id application but I agree that manually edit a track is very hard. Need to improve that.

**Comment by corentin87** [comment]
Is this plan anytime soon?  I am doing track annotation atm and it is indeed pretty difficult to work with import annotation since all frames are key frames.

**Comment by drabaioli** [comment]
Any update on this?

**Comment by Quakumei** [comment]
Any update on this?

**Comment by Quakumei** [comment]
Indeed, if you annotate attributes on an uploaded track annotation with keyframes is gonna be painful because you need to switch attributes on every single frame. Let's name it Track 1.  What you can do instead is create a new track, Track 2 (e.g. by copying Track 1 at its first frame), switch outside property of Track 1 on the first frame, and merge these two tracks by getting Track 3. Track 3 will not have problems with attribute annotation and will indeed pass on attributes you set to the next frames. 

Kinda of a hack but that works
